test/pkg/environment/aws/setup.go

```diff
@@ -34,23 +31,18 @@ var (
 	}
 )
 
-func (env *Environment) BeforeEach(opts ...common.Option) {
+func (env *Environment) BeforeEach() {
 	persistedSettings = env.ExpectSettings()
-	env.Environment.BeforeEach(opts...)
+	env.Environment.BeforeEach()
 }
 
-func (env *Environment) Cleanup(opts ...common.Option) {
-	options := common.ResolveOptions(opts)
-	if !options.DisableDebug {
-		fmt.Println("------- START AWS CLEANUP -------")
-		defer fmt.Println("------- END AWS CLEANUP -------")
```

Comment:
Why remove these? Is it because debug is always enabled now?

Reply:
Yes, WDYT about using `ginkgo.vv` instead of relying on our own mechanism for printing the stages of the test?

Reply:
Discussed offline and were fine with removing these lines in favor of `ginkgo.vv`